roles/rsyslog/roles/input_roles/basics/tasks/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: Set basics facts
   set_fact:
-    rsyslog_role_packages: rsyslog-gnutls
+    rsyslog_role_packages: "{{ rsyslog_basics_packages }}"
```

Comment:
shouldn't it be called just `rsyslog_packages`? Why the `role` in the name? Also, why not drop the `set_fact` and rename `rsyslog_basics_packages` to `rsyslog_packages` (or `__rsyslog_packages` if the variable is supposed to be internal, and I suspect it is)?

Reply:
The var rsyslog_role_packages is used here [1].  They need to match.  The current rsyslog_role_packages could be set in each input and output sub-roles.  So, I think we should replace rsyslog_role_packages with __rsyslog_packages to make it internal, and replace each rsyslog_something_packages with __rsyslog_something_packages?
[1] https://github.com/linux-system-roles/logging/blob/master/roles/rsyslog/tasks/deploy.yaml#L2-L8

Reply:
I modified the code like this
https://github.com/linux-system-roles/logging/pull/87/files#diff-532de15639f14a0202b8cd2880a8bd4cR4-R10
with the comment:
- To call include_role, instead of setting variables to pass with set_fact, put them in the block with vars to make them local in the scope.

Please note that I made the same change to all the include_role.

Reply:
You don't need a block. Put vars: directly on the include_role statement.

Reply:
Thanks, @pcahyna.  Fixed.